dashboard/app/controllers/plc/user_course_enrollments_controller.rb

```diff
@@ -20,17 +11,14 @@ def new
   # POST /plc/user_course_enrollments.json
   def create
     if @user_course_enrollment.save
-      redirect_to @user_course_enrollment, notice: 'User course enrollment was successfully created.'
+      redirect_to action: :course_view
     else
       redirect_to action: :new
     end
   end
 
-  # DELETE /plc/user_course_enrollments/1
-  # DELETE /plc/user_course_enrollments/1.json
-  def destroy
-    @user_course_enrollment.destroy
-    redirect_to action: :index
+  def course_view
+    @enrollments = current_user.plc_enrollments
```

Comment:
Prefer using the cancancan-provided scope, e.g:
```ruby
@enrollments = @enrollments.where(user: current_user)
```

The SQL generated is the same, and this feels more rails-y.
```sql
SELECT `plc_user_course_enrollments`.* FROM `plc_user_course_enrollments` WHERE `plc_user_course_enrollments`.`user_id` = 1
```